src/linter/linter.ts

```diff
@@ -1,21 +1,70 @@
 import {graphFromObject} from "@ui5/project/graph";
-import {createReader, createWorkspace, createReaderCollection} from "@ui5/fs/resourceFactory";
+import {createReader, createWorkspace, createReaderCollection, createFilterReader} from "@ui5/fs/resourceFactory";
 import {FilePath, LinterOptions, LintResult} from "./LinterContext.js";
 import lintWorkspace from "./lintWorkspace.js";
 import {taskStart} from "../utils/perf.js";
 import path from "node:path";
 import posixPath from "node:path/posix";
 import {stat} from "node:fs/promises";
 import {ProjectGraph} from "@ui5/project";
-import {AbstractReader} from "@ui5/fs";
+import type {AbstractReader, Resource} from "@ui5/fs";
+import ConfigManager from "../utils/ConfigManager.js";
+import {minimatch} from "minimatch";
 
 async function lint(
 	resourceReader: AbstractReader, options: LinterOptions
 ): Promise<LintResult[]> {
 	const lintEnd = taskStart("Linting");
+	let {ignorePattern} = options;
 
-	const workspace = createWorkspace({
+	const configMngr = new ConfigManager();
+	const config = await configMngr.getConfiguration();
+	ignorePattern = [
+		...(config.ignores ?? []),
+		...(ignorePattern ?? []), // CLI patterns go after config patterns
+	].filter(($) => $);
+
+	let fsBasePath = "";
+	let fsBasePathTest = "";
+	let virBasePath = "/resources/";
+	let virBasePathTest = "/test-resources/";
+	try {
+		const graph = await getProjectGraph(options.rootDir);
+		const project = graph.getRoot();
+		fsBasePath = project.getSourcePath();
+		fsBasePathTest = path.join(project.getRootPath(), project._testPath);
+
+		if (!project._isSourceNamespaced) {
+			// Ensure the virtual filesystem includes the project namespace to allow relative imports
+			// of framework resources from the project
+			virBasePath += project.getNamespace() + "/";
+			virBasePathTest += project.getNamespace() + "/";
+		}
+	} catch {
+		// Project is not resolved i.e. in tests
+	}
+
+	const relFsBasePath = path.relative(options.rootDir, fsBasePath);
+	const relFsBasePathTest = fsBasePathTest ? path.relative(options.rootDir, fsBasePathTest) : undefined;
+
+	const filteredCollection = createFilterReader({
 		reader: resourceReader,
+		callback: (resource: Resource) => {
+			if (!ignorePattern?.length) {
+				return true;
+			}
+
+			// Minimatch works with FS and relative paths.
+			// So, we need to convert virtual paths to fs
+			const resPath = transformVirtualPathToFilePath(
```

Comment:
this mapping doesn't consider the project namespace. It maps from `/resources/sap/ui/demo/todo/view/App.view.xml` to `webapp/sap/ui/demo/todo/view/App.view.xml`, which is not correct.

Reply:
Ah, thanks!
Now, it's fixed